spec/rubocop/cop/performance/reverse_each_spec.rb

```diff
@@ -12,6 +12,8 @@
     expect_offense(<<~RUBY)
       array&.reverse.each { |e| puts e }
              ^^^^^^^^^^^^ Use `reverse_each` instead of `reverse.each`.
+      array&.reverse&.each { |e| puts e }
+             ^^^^^^^^^^^^^ Use `reverse_each` instead of `reverse.each`.
```

Comment:
Please separate this, an independent test instead of mixing it in the existing `it` block.

Reply:
Hate inventing new names for tests 😅 Updated